runtime/litentry/Cargo.toml

```diff
@@ -95,6 +95,17 @@ pallet-extrinsic-filter = { path = "../../pallets/extrinsic-filter", default-fea
 pallet-parachain-staking = { path = "../../pallets/parachain-staking", default-features = false }
 runtime-common = { path = '../common', default-features = false }
 
+# EVM/Frontier
+fp-rpc = { git = "https://github.com/paritytech/frontier", branch = "polkadot-v0.9.39", default-features = false }
+fp-self-contained = { git = "https://github.com/paritytech/frontier", branch = "polkadot-v0.9.39", default-features = false }
+pallet-ethereum = { git = "https://github.com/paritytech/frontier", branch = "polkadot-v0.9.39", default-features = false, features = ["forbid-evm-reentrancy"] }
+pallet-evm = { git = "https://github.com/paritytech/frontier", branch = "polkadot-v0.9.39", default-features = false, features = ["forbid-evm-reentrancy"] }
+
+# Moonbeam
+moonbeam-evm-tracer = { git = "https://github.com/litentry/astar-frame", branch = "polkadot-v0.9.39", default-features = false }
+moonbeam-rpc-primitives-debug = { git = "https://github.com/litentry/astar-frame", branch = "polkadot-v0.9.39", default-features = false }
+moonbeam-rpc-primitives-txpool = { git = "https://github.com/litentry/astar-frame", branch = "polkadot-v0.9.39", default-features = false }
```

Comment:
Don't we want to onboard them to rococo first?

Reply:
Yes. For fake compiling. These pallet imports are for mimicing a fake RPC for code consistency. Their main code are never actually compiled in runtime. Only RPC require some of their traits.

Reply:
Wait a minute. Let me check if we still need fake implementation after the separating of node code.

Reply:
You are right. Code removed now.